README.md

```diff
@@ -51,6 +51,10 @@ Additional goals for Wasmtime include:
 [Cranelift]: https://github.com/bytecodealliance/cranelift
 [Lightbeam]: https://github.com/bytecodealliance/wasmtime/tree/master/crates/lightbeam
 
+## Using Wasmtime
+
+There are two primary ways to use Wasmtime: as a standalone runtime for running WASI-compatible WASM binaries, and via the embedding API in Rust and C.
```

Comment:
As a very mild nit, WASI is an acronym, but wasm is not, so it's "WebAssembly", "wasm", or "Wasm", but not "WASM".